Models/Departments.py

```diff
@@ -0,0 +1,12 @@
+from sqlalchemy.ext.declarative import declarative_base
+from sqlalchemy import Column, Integer, String
+
+Base = declarative_base()
```

Comment:
...Does this work? I mean, you're creating a **different* base for each model. Why don't you create a common `models/__init__.py` which declares the base, and then `from . import Base` in each model file?

Also, why even bother breaking them up into multiple files? Seems premature to me.